src/main/java/org/broadinstitute/dsm/model/participant/data/ParticipantData.java

```diff
@@ -74,24 +73,29 @@ public static NewParticipantData parseDto(@NonNull ParticipantDataDto participan
         return participantData;
     }
 
-    public Map<String, String> mergeParticipantData(@NonNull AddFamilyMemberPayload familyMemberPayload) {
+    public void setFamilyMemberData(@NonNull AddFamilyMemberPayload familyMemberPayload) {
         FamilyMemberDetails familyMemberData =
                 familyMemberPayload.getData().orElseThrow(() -> new NoSuchElementException("Family member data is not provided"));
         familyMemberData.setCollaboratorParticipantId(
                 familyMemberPayload.getRealm().orElse("").toUpperCase() +
```

Comment:
there is a collaborator_id_prefix in ddp_instance table which could be used for this. For RGP it is `RGP` and for CMI studies it is `[NAME]Project` in case other studies want to use it

Reply:
maybe worth changing, but we could also do that at a later point and leave it as it is for now

Reply:
I like that idea actually, since it will provide more dynamicity, will check if it is not big deal to do it quickly

Reply:
👍